intl/msg_hash_us.h

```diff
@@ -1597,13 +1597,13 @@ MSG_HASH(MENU_ENUM_SUBLABEL_CPU_CORES,
 MSG_HASH(MENU_ENUM_SUBLABEL_FPS_SHOW,
       "Displays the current framerate per second onscreen.")
 MSG_HASH(MENU_ENUM_SUBLABEL_INPUT_HOTKEY_BINDS,
-      "Configure hotkey settings.")
+      "Change settings for hotkey.")
```

Comment:
To stay consistent with others.
Change settings for video output.
Change settings for audio output.
Change settings for joypads, keyboard, and mouse.
etc...